Library/Homebrew/service.rb

```diff
@@ -538,7 +538,7 @@ def serialize
         linux: (service_name if service_name != default_service_name),
       }.compact
 
-      unless command?
+      if @run_params.blank?
         return name_params.blank? ? {} : { name: name_params }
       end
```

Comment:
```suggestion
      return { name: name_params }.compact if @run_params.blank?
```
Maybe?

Reply:
I'm tempted to error if `name_params` is blank. An empty service doesn't sound valid to me.

Reply:
Throwing an exception sounds reasonable to me. But CC @apainintheneck

Reply:
Ah some tests rely on this so I'll just do what you suggested for now to merge the breakage quickly.

Reply:
Failing here probably makes sense.